cli/src/funTest/kotlin/ExamplesFunTest.kt

```diff
@@ -134,7 +134,7 @@ class ExamplesFunTest : StringSpec() {
         }
 
         "example.rules.kts can be compiled and executed" {
-            val resultFile = File("src/funTest/assets/semver4j-analyzer-result.yml")
+            val resultFile = File("src/funTest/assets/semver4j-advisor-result.yml")
```

Comment:
As the scanner could also be run after the advisor, using "advisor" does not really make clear that a scanner result is included. Maybe be super-explicit and name it `semver4j-analyzer-scanner-advisor-result.yml`, or just `semver4j-ort-result.yml`?

Reply:
I think it would be valuable to assume an order, at least for the test assets in order to not making our lives unnecessarily hard finding names for files. Changed according to your comment regardlessly.